src/test/java/com/redhat/devtools/lsp4ij/LSPIJUtils_toOffset_serverReportsOffsetOnlyPositionsTest.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package com.redhat.devtools.lsp4ij;
+
+import com.intellij.openapi.editor.Document;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiFile;
+import com.intellij.util.containers.ContainerUtil;
+import com.redhat.devtools.lsp4ij.fixtures.LSPCodeInsightFixtureTestCase;
+import com.redhat.devtools.lsp4ij.server.definition.ClientConfigurableLanguageServerDefinition;
+import com.redhat.devtools.lsp4ij.server.definition.LanguageServerDefinition;
+import com.redhat.devtools.lsp4ij.server.definition.launching.ClientConfigurationSettings;
+import org.eclipse.lsp4j.Position;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Tests for {@link LSPIJUtils#toOffset(Position, Document)} when
+ * {@link ClientConfigurationSettings#serverReportsOffsetOnlyPositions} is enabled.
+ */
+public class LSPIJUtils_toOffset_serverReportsOffsetOnlyPositionsTest extends LSPCodeInsightFixtureTestCase {
+
+    public LSPIJUtils_toOffset_serverReportsOffsetOnlyPositionsTest() {
+        super("*.txt");
+        setClientConfigurable(true);
+    }
+
+    @Override
+    protected void setUp() throws Exception {
```

Comment:
Need to get the language server to get the client config to enable the feature flag.